### PR TITLE
Change policies

### PR DESCRIPTION
**Pull request by DFurt**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -56,6 +37,53 @@ EOF
   containers = concat([local.container], var.additional_containers)
 }
 
+resource "aws_ecr_lifecycle_policy" "ramp_ecr_lifecycle_policy" {
```
Comment (MiguelNdeCarvalho) [inline main.tf:40]: Change the resource name from `ramp_ecr_lifecycle_policy` to `ecr_lifecycle_policy` only

File: variables.tf [hunk 2]
```diff
@@ -56,12 +56,23 @@ variable "ecr_scan_on_push" {
...
   description = "Controls how many images should be kept in the ECR repo."
 }
 
+variable "ecr_number_of_images_to_keep_on_branches" {
+  type        = number
+  default     = 10
+  description = "Controls how many images should be kept on branches in the ECR repo."
+}
+
+variable "ecr_number_of_images_to_keep_untagged" {
```
Comment (MiguelNdeCarvalho) [inline variables.tf:71]: Should we change the variable name to `ecr_number_of_images_untagged` to be more concise and shorter?

File: variables.tf [hunk 3]
```diff
@@ -56,12 +56,23 @@ variable "ecr_scan_on_push" {
...
 }
 
-variable "ecr_number_of_images_to_keep" {
+variable "ecr_number_of_images_to_keep_on_main" {
   type        = number
   default     = 30
   description = "Controls how many images should be kept in the ECR repo."
 }
 
+variable "ecr_number_of_images_to_keep_on_branches" {
```
Comment (MiguelNdeCarvalho) [inline variables.tf:65]: Should we change the variable name to `ecr_number_of_images_from_branches` to be more concise and a bit shorter?

File: variables.tf [hunk 4]
```diff
@@ -56,12 +56,23 @@ variable "ecr_scan_on_push" {
   description = "Controls if ECR should scan images after pushed."
 }
 
-variable "ecr_number_of_images_to_keep" {
+variable "ecr_number_of_images_to_keep_on_main" {
```
Comment (MiguelNdeCarvalho) [inline variables.tf:59]: Should we change the variable name to `ecr_number_of_images_from_main` to be more concise and a bit shorter?

**Review by MiguelNdeCarvalho — CHANGES_REQUESTED** [review]
I have proposed some changes, let me know what you think about them!

File: main.tf (hunk 1 above)
Comment (DFurt) [reply main.tf]: Done

File: variables.tf (hunk 2 above)
Comment (DFurt) [reply variables.tf]: Done

File: variables.tf (hunk 4 above)
Comment (DFurt) [reply variables.tf]: Done

**Comment by DFurt** [comment]
I have changed the requested parameters to suggested ones

**Pull request closed without merging**